__tests__/integration/melt-tokens.test.js

```diff
@@ -305,19 +306,53 @@ describe('melt tokens', () => {
       .send({
         token: tokenA.uid,
         address: await wallet1.getAddressAt(1),
-        amount: 150
+        amount: 100
       })
       .set({ 'x-wallet-id': wallet1.walletId });
 
     expect(response.body.success).toBe(true);
 
     await TestUtils.pauseForWsUpdate();
 
+    const balance1htr = await wallet1.getBalance();
+    const balance1tka = await wallet1.getBalance(tokenA.uid);
+    expect(balance1htr.available).toBe(9);
+    expect(balance1tka.available).toBe(50);
+
+    done();
+  });
+
+  it('should melt and send melt output to the correct address', async done => {
+    const address0 = await wallet1.getAddressAt(0);
+    const response = await TestUtils.request
+      .post('/wallet/melt-tokens')
+      .send({
+        token: tokenA.uid,
+        address: await wallet1.getAddressAt(16),
+        melt_authority_address: address0,
+        amount: 50
+      })
+      .set({ 'x-wallet-id': wallet1.walletId });
+
+    const transaction = response.body;
+    expect(transaction.success).toBe(true);
+    await TestUtils.pauseForWsUpdate();
+
     const balance1htr = await wallet1.getBalance();
     const balance1tka = await wallet1.getBalance(tokenA.uid);
     expect(balance1htr.available).toBe(9);
     expect(balance1tka.available).toBe(0);
 
+    // Validating a new melt authority was created by default
+    const authorityOutputs = transaction.outputs.filter(
+      o => transaction.isTokenDataAuthority(o.tokenData)
```

Comment:
This is not present on the most recent version of the lib.

I noticed we have another PR that will implement this feature in the dev branch.
Are we planning to have this merged later or will we make a force push to the master branch once the dev branch is ready to be released?